web/src/Reply.css

```diff
@@ -4,3 +4,11 @@
   bottom: 0;
   box-shadow: 0 2px 20px rgba(0, 0, 0, 0.35);
 }
+
+.fr .link-btn {
```

Comment:
The question is, do all browsers use this color set?

Reply:
Yes, this color comes from the old css